src/js/components/Ballot/EmailBallotToFriendsModal.jsx

```diff
@@ -21,6 +21,12 @@ export default class EmailBallotToFriendsModal extends Component {
 
   constructor (props) {
     super(props);
+    let ballotLink = "";
+    if (this.props.ballot_link) {
+      ballotLink = web_app_config.WE_VOTE_URL_PROTOCOL + web_app_config.WE_VOTE_HOSTNAME + this.props.ballot_link;
+    } else {
```

Comment:
Hello @armendo, in general we are moving away from defining state values in the constructor. We declare them there (this.state = {}), but don't give them complex values. Instead we assign the values in componentDidMount and componentWillReceiveProps.